### PR TITLE
Update  doc, license and add Dockerfile

### LINKED ISSUE #7: Cleanup
Updated documentation to install on windows with cuda support.  

Dockerfile for docker image creation.

### PR DESCRIPTION
**Pull request by mpelchat04**
Update  documentation for installation  
Update License with Open Government License - Canada  
Add Dockerfile  
Closes #7

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ARG CONDA_DIR=/opt/conda
+ARG USERNAME=gdl_user
+ARG USERID=1000
+
+
+# Install Mamba directly
+ENV PATH=$CONDA_DIR/bin:$PATH
+
+# RNCAN certificate; uncomment (with right .cer name) if you are building behind a FW
+COPY NRCan-RootCA.cer /usr/local/share/ca-certificates/cert.crt
```
Comment (valhassan) [inline Dockerfile]: Are these commands not specific to NRCAN environments?

**Review by valhassan — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# syntax=docker/dockerfile:1
+FROM nvidia/cuda:12.4.1-cudnn-runtime-ubuntu20.04
```
Comment (jfbourgon) [inline Dockerfile:2]: we should consider migrating to ubuntu22 (or even ubuntu24) as 20.04 LTS Standard security maintance is target to end in Apr 2025: https://ubuntu.com/about/release-cycle

File: Dockerfile (hunk 1 above)
Comment (mpelchat04) [reply Dockerfile]: You're right, I commented the lines related to the certificate thing.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+USER $USERNAME
+WORKDIR /home/$USERNAME/
+COPY requirements.txt . /home/$USERNAME/
+RUN cd /home/$USERNAME \
+    && conda config --set ssl_verify no \
+    && mamba create --name geo-inference \ 
+    && mamba install pip \
+    && pip install --upgrade pip \
+    && pip install --no-cache-dir git+https://github.com/NRCan/geo-inference.git \
+    && pip install --no-cache-dir -r /home/$USERNAME/requirements.txt \
```
[the remark this answers is not in the document]
Comment (mpelchat04) [reply Dockerfile]: Removed.

File: Dockerfile (hunk 2 above)
Comment (mpelchat04) [reply Dockerfile]: Migrated to ubuntu22.04

**Pull request merged**